third_party/gtp_ovs/ovs-gtp-patches/2.15/build.sh

```diff
@@ -1,10 +1,22 @@
 #! /bin/bash
+#
+# Copyright 2022 The Magma Authors.
```

Comment:
Here I am unsure: should this be 2022 or the year the script was originally created?